### PR TITLE
Revert calls to `queueRepositoryUpdateAndModelRefresh`

### LINKED ISSUE #1465: Does `queueRepositoryUpdateAndModelRefresh` need to be called from so many places?
Can't we generally just rely on listening to changes to .git/machete and git repository? 🤔

### PR DESCRIPTION
**Pull request by PawelLipski**
☹️

### CODE REVIEW

**Comment by PawelLipski** [comment]
Lemme install & test that locally first for a moment...

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/DiscoverAction.java [hunk 1]
```diff
@@ -132,6 +136,7 @@ public void run(ProgressIndicator indicator) {
       @Override
       @UIEffect
       public void onSuccess() {
+        baseEnhancedGraphTable.queueRepositoryUpdateAndModelRefresh();
```
Comment (mkondratek) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/DiscoverAction.java:139]: here needed too?

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/CheckoutAndExecute.java [hunk 2]
```diff
@@ -10,25 +10,27 @@
...
+import com.virtuslab.gitmachete.frontend.ui.api.table.BaseEnhancedGraphTable;
 import com.virtuslab.qual.async.ContinuesInBackground;
 
 @CustomLog
 final class CheckoutAndExecute {
   private CheckoutAndExecute() {}
 
   @ContinuesInBackground
-  static void checkoutAndExecuteOnUIThread(GitRepository gitRepository, String branchName,
+  static void checkoutAndExecuteOnUIThread(GitRepository gitRepository, BaseEnhancedGraphTable graphTable, String branchName,
```
Comment (mkondratek) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/CheckoutAndExecute.java:21]: 😢

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/SlideOut.java [hunk 3]
```diff
@@ -116,14 +120,15 @@ private void suggestBranchDeletion(String branchName, @UI Runnable doInUIThreadW
...
     if (shouldDelete) {
-      GitBrancher.getInstance(project).deleteBranches(Collections.singletonMap(branchName,
-          Collections.singletonList(gitRepository)), () -> {
-            VcsNotifier.getInstance(project).notifySuccess(/* displayId */ null,
-                /* title */ "",
-                getString("action.GitMachete.BaseSlideOutAction.notification.title.slide-out-success.with-delete.HTML").fmt(
-                    branchName));
-            doInUIThreadWhenReady.run();
-          });
+      graphTable.queueRepositoryUpdateAndModelRefresh(
```
Comment (mkondratek) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/SlideOut.java:123]: yeah, makes sense definitely here

**Review by mkondratek — APPROVED** [review]

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/DiscoverAction.java (hunk 1 above)
Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/DiscoverAction.java:139]: Right... likely redundant here, lemme check...

**Pull request merged**